backend/src/main/java/io/debezium/configserver/rest/ConnectorResource.java

```diff
@@ -23,13 +32,21 @@
 import io.debezium.configserver.model.ConnectorType;
 import io.debezium.configserver.model.FilterValidationResult;
 import io.debezium.configserver.model.PropertiesValidationResult;
+import io.debezium.configserver.rest.client.KafkaConnectClient;
 import io.debezium.configserver.rest.model.BadRequestResponse;
 import io.debezium.configserver.rest.model.ServerError;
 import io.debezium.configserver.service.ConnectorIntegrator;
 
 @Path("/api")
 public class ConnectorResource {
 
+    public static final String PROPERTY_KAFKA_CONNECT_URI = "kafka.connect.uri";
+
+    private static final Logger LOG = Logger.getLogger(ConnectorResource.class);
+
+    @ConfigProperty(name = PROPERTY_KAFKA_CONNECT_URI, defaultValue = "http://localhost:8083")
+    String kafkaConnectBaseUri = "http://localhost:8083";
```

Comment:
I think this can be injected as a `List<URI`>.

Reply:
`Caused by: java.lang.RuntimeException: Error injecting java.util.List&lt;java.net.URI&gt; io.debezium.configserver.rest.ConnectorResource.kafkaConnectBaseUris` when using a comma-separated list.

Reply:
Should we allow whitespaces in the URI list? then we need to to it manually, but we can do `List<String>`. @gunnarmorling

Reply:
Let's disallow whitespaces then. Worth double-checking on quarkus-dev though whether that's a known limitation for List<URI> or whether it's a bug we should log.

Reply:
Not really a bug. `List<>` injection from a config property will split on comma. And then it leaves all fields untouched. but a URI with `    http://localhost:123    ` is just wrong because it's not allowed to start or end URI with whitespaces.
I addressed that in the last or second last commit with making it a `List<String>`. Can you live with that? Imo it makes the config more accessible to users.

Reply:
Sure, works for me. Still not sure though why Quarkus couldn't trim comma-separated URI values?

Reply:
I guess there's just no specific handling for `List<URI>`, but that's just a guess. It's risky to manipulate the input and so they don't trim it, also it is safe for URI.